### PR TITLE
Aditya

### PR DESCRIPTION
**Pull request by adityajatiya**

### CODE REVIEW

File: Basic/print/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HelloWorld {
+    public static void main(String[] args) {
+        System.out.println("Hello world");
+    }
+}
```
Comment (Madhur-Jatiya) [inline Basic/print/HelloWorld.java]: add empty line below

File: Basic/print/NamasteIndia.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class NamasteIndia{
+    public static void main(String[] args){
+        System.out.println("Namste India");
+    }
+}
```
Comment (Madhur-Jatiya) [inline Basic/print/NamasteIndia.java:5]: add empty line below

File: Basic/print/HelloWorld.java (hunk 1 above)
Comment (adityajatiya) [reply Basic/print/HelloWorld.java]: fix

File: Basic/print/NamasteIndia.java (hunk 2 above)
Comment (adityajatiya) [reply Basic/print/NamasteIndia.java:5]: fix

**Pull request merged**